packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -179,7 +183,7 @@ export class ProjectService {
         let explores: (Explore | ExploreError)[];
         try {
             await adapter.test();
-            explores = await adapter.compileAllExplores();
+            explores = await adapter.compileAllExplores(projectUuid);
```

Comment:
from our discussion at standup, let's remove this line, so all we do is test the adapter and stop